tests/h/schemas/annotation_test.py

```diff
@@ -722,6 +724,36 @@ def test_raises_if_invalid_offset(self, schema, offset):
         with pytest.raises(ValidationError):
             validate_query_params(schema, input_params)
 
+    def test_raises_if_invalid_search_after_date(self, schema):
+        input_params = NestedMultiDict(MultiDict({"search_after": "invalid_date"}))
+
+        with pytest.raises(ValidationError):
+            validate_query_params(schema, input_params)
+
+    @pytest.mark.parametrize('search_after,sort', (
+        ("2009-02-16", "updated"),
+        ("2018-01-01T02:26:01.03-00:08", "created"),
+        ("5045000300.9", "updated"),
+        ("8273640", "id")))
```

Comment:
I think it would be good to include at least one example here which exactly matches the format of times in API responses - since this should be the most common source of input values for this parameter.

Reply:
I added test cases with the order of magnitude specified as our api responses return. I think it's important to note that elasticsearch doesn't go down to that order of magnitude and therefore those extra digits are meaningless but it doesn't fail and still parses the date as expected which I think is what you were after here.